### DISCUSSION TITLE
facing issue in new symfony version 5.4

### DISCUSSION THREAD

**Discussion by Thejaswini-Rao-U** [symfony/symfony#51353] [Q&A]
### Symfony version(s) affected

5.4

### Description

Getting error of

**"Cannot retrieve the container from a non-booted kernel."**
this error showing in the line WebsiteSetting::getByName('SAML_ENV');

### How to reproduce

Im using PHP 8.1, Symfony 5.4, Pimcore 10.6.7

When I'm calling WebsiteSetting::getByName('SAML_ENV');

gets error in my dependency injection file, which is having method

public function prepend(ContainerBuilder $container){
   WebsiteSetting::getByName('SAML_ENV');
}

Getting error of

**"Cannot retrieve the container from a non-booted kernel."**
this error showing in the line WebsiteSetting::getByName('SAML_ENV');

please help to solve this issue

Thanks in advance

**Comment by xabbuh** [comment]
If you think that you found a bug in the code, please provide a small example application that allows to reproduce your issue.

For support, please refer to one of the [support channels](https://symfony.com/support).

**Comment by Thejaswini-Rao-U** [comment]
@xabbuh , Thanks for the response, i cannot provide the code

but seeing error is it possible to find out what causes the issue?

even i get issue of default not registered as service or class, after updating to symfony 5.4

![ntofiicationPathError](https://github.com/symfony/symfony/assets/78895325/90c0e625-0422-40f6-8f71-737dd71a2faa)
![azure error](https://github.com/symfony/symfony/assets/78895325/f2ea51da-253b-4107-8dce-7f33382f6f1f)
